### ISSUE TITLE
Editable children do not save their state in the tree.

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot#68553]
### Godot version

v3.5.1.rc1.official [293c3844b]

### System information

w10 64

### Issue description

When I activate the "editable children" option on a node the node expands. Ok, so I edit the children and collapse the node to not take up space in the tree.
I close the project, I reopen the project and the node appears expanded, I have to contract the node again so as not to take up space in the tree.

Problem:
Nodes with "editable children" do not save their "expand/collapse" state.
This is annoying because the user must collapse the node (if he wants to because for example, as in my case, it takes up a lot of space in the tree) every time he opens the project because the node does not save its "expanded/collapsed" state.

In this video you can see the following:
- I open the project and the node is shown with the editable children expanded.
- I contract the node.
- I reload the project.
- The node appears expanded again.

https://youtu.be/VlFhdG-6Tu4

### Steps to reproduce

Create a node that inherits from another and make its children editable.
The node expands showing its children.
Collapse the node.
Save the project.
Load the project again.
Check that the state of the node is not saved and is shown expanded again.

### Minimal reproduction project

...

**Comment by adamscott** [comment]
Please upload your video on Github directly, as it helps against webrot.

**Comment by adamscott** [comment]
Could not reproduce on 3.5.1
Please test on the latest versions of Godot before creating an issue.

**Comment by Torguen** [comment]
The link works.
Github has a 10 meg file limit, I tried to upload my file and it exceeded that limit.

I can't reproduce it in an example project either but it happens in my project.

I can send the project by private.

I haven't read any rules that require testing for problems in the latest versions of Godot.

**Comment by adamscott** [comment]
> I haven't read any rules that require testing for problems in the latest versions of Godot.

Because the issue becomes obsolete if it was fixed on a newer release and the ticket will be closed.

**Comment by Torguen** [comment]
I only use official published versions, the latest official published version is:
v3.5.1.rc1.official [293c3844b]

Maybe it's not fixed, you can't reproduce it because it only happens in projects like mine, which meet some condition that causes the error.

Maybe you tried to reproduce it in a small project and, as I said before, in a small project it doesn't happen.

**Comment by adamscott** [comment]
> I only use official published versions, the latest official published version is:
> v3.5.1.rc1.official [[293c384](https://github.com/godotengine/godot/commit/293c3844b3424b4a64d6245f99145266101a146f)]

No. [The latest published version is 3.5.1](https://godotengine.org/article/maintenance-release-godot-3-5-1)

![Screenshot_20221113-100544](https://user-images.githubusercontent.com/270928/201528779-4e9bb2d7-78b1-4a6a-b468-79ba26f0b45b.png)

**Comment by Torguen** [comment]
Ok, I missed that version!

Also the problem is different, now the node will always appear collapsed even if you expand it, it will not keep its state and it will appear collapsed.

**Comment by clayjohn** [comment]
I'm not sure if there is anything actionable left here. 

If the problem is only in a 3.5.1 release candidate but is fixed in 3.5.1 then can we go ahead and close this bug report?

**Comment by Torguen** [comment]
The problem is that the nodes do not save their "expanded/collapsed" state in the tree when the project is closed.
v3.5.1.stable.official [6fed1ffa3]

**Comment by adamscott** [comment]
Please join a MRP. If it's not reproducible easily, the problem is maybe something else.

**Comment by Torguen** [comment]
This keeps happening in v3.5.2.rc2.official [1a2bf3eb4]

Watch the video:

https://user-images.githubusercontent.com/19652075/222676493-26973a4e-6abf-4b4b-8a80-520154eaf6f5.mp4

**Comment by Torguen** [comment]
The error does not occur if you make any changes to the node and the scene needs to be saved.
Follow these steps so you don't make the mistake and the node saves collapsed.
Collapse the node.
Change the name of the node.
Save the scene.
Now the node will appear collapsed but that doesn't fix the problem.

Conclusion:
The node does not save its collapsed/collapsed state when closing the scene.
The node only saves its collapsed/collapsed state if it makes any changes that require the scene to be saved.
Otherwise, the node does not save its collapsed/collapsed state.

**Comment by Torguen** [comment]
I don't understand why you need a test, this happens in all operating systems, it's a Godot thing, it's a Godot save scene thing.
